### ISSUE TITLE
Not converging!

### ISSUE THREAD

**Issue by astorfi** [srviest/char-cnn-pytorch#3]
I have been able to run the model as is but it simply not converging!
For how many epochs I should run the model?

**Comment by astorfi** [comment]
That's what I got:

Epoch[30] Batch[2400] - loss: 1.386194  lr: 0.00050  acc: 25.000% (8/32)
Evaluation - loss: 1.386315  lr: 0.00050  acc: 24.960% (1893/7584) 
Label:   0	Prec:   0.0% (0/0)        Recall:   0.0% (0/1895)     F-Score:   N/A
Label:   1	Prec:  25.0% (1893/7584)  Recall: 100.0% (1893/1893)  F-Score:  39.9%
Label:   2	Prec:   0.0% (0/0)        Recall:   0.0% (0/1898)     F-Score:   N/A
Label:   3	Prec:   0.0% (0/0)        Recall:   0.0% (0/1898)     F-Score:   N/A

No convergence for training and so no good results for evaluation.
I run it as is.

**Comment by srviest** [comment]
This is probably due to bad initialization, try reducing the learning rate.

**Comment by astorfi** [comment]
Yes, you were right. It's solved now. Thank you so much.
